llvm/test/TableGen/def-multiple-operands.td

```diff
@@ -0,0 +1,33 @@
+// RUN: llvm-tblgen -gen-instr-info -I %p/../../include %s | FileCheck %s
+
```

Comment:
Can you add a short description of what this test is checking?

Reply:
Done, this test checks whether the `MCInstrDesc` entry for `InstA` has the expected `NumOperands` and `NumDefs`. It should have 3 defs out of 4 operands.